website/www/site/content/en/get-started/quickstart-py.md

```diff
@@ -23,7 +23,7 @@ If you're interested in contributing to the Apache Beam Python codebase, see the
 
 {{< toc >}}
 
-The Python SDK supports Python 3.8, 3.9, 3.10 and 3.11. Beam 2.48.0 was the last release with support for Python 3.7.
+The Python SDK supports Python 3.8, 3.9, 3.10, 3.11 and 3.12. Beam 2.48.0 was the last release with support for Python 3.7.
```

Comment:
Should we also drop 3.8 from this list? According to the `tox.ini` [file](https://github.com/apache/beam/blob/master/sdks/python/tox.ini#L20), 3.8 is no longer tested.

Reply:
Fine for now and we should do this after Beam 2.61.0 release is out. Beam releases every 6 weeks while the website/ is deployed on daily bases cc: @jrmccluskey